tabpy-tools/tabpy_tools/rest.py

```diff
@@ -3,11 +3,11 @@
 import requests
 from requests.auth import HTTPBasicAuth
 from re import compile
+import json
```

Comment:
do we need both? simplejson in Python3 is just a wrapper for json for backward compatibility

Reply:
We reference both on line 27 & 28. I was just cleaning up line 9 from the old file.

Want me to refactor?

Reply:
Depends on how much work it is. Either refactor or open an issue (and TFS bug) for it.

Reply:
no need to open a bug, removed simplejson in the latest commit